README.md

```diff
@@ -45,6 +48,28 @@ use {
 }
 ```
 
+Using [lazy.nvim](https://github.com/folke/lazy.nvim)
+
+```lua
+-- init.lua:
+{
+    "ThePrimeagen/harpoon",
+    branch = "harpoon2",
+    dependencies = { "nvim-lua/plenary.nvim" },
+    opts = {},
+    config = function()
+        local harpoon = require("harpoon")
```

Comment:
It's missing `harpoon:setup()`

I don't if Prime agree with this, but I think it's better to add harpoon configuration in folder `~/.config/nvim/lua/plugins/`, but maybe this is optional, more important is harpoon:setup

Reply:
Thank you for taking the time to review my PR!
I've added the `harpoon:setup()` call as suggested. I also agree that the config belongs in its own file in `~/.config/nvim/lua/plugins/`, so I've updated the filepath and added a return statement to reflect this.